src/Check/ScrutinizerCheck.php

```diff
@@ -7,7 +7,9 @@
 
 class ScrutinizerCheck extends Check
 {
+    // phpcs:disable
     const THRESHOLD = 6.5;
+    // phpcs:enable
```

Comment:
```suggestion
    const THRESHOLD = 6.5;
```
The PHPCS issue has been resolved in another PR, which has since been merged.